server/on-demand-entry-handler.js

```diff
@@ -5,9 +5,9 @@ import { parse } from 'url'
 import resolvePath from './resolve'
 import touch from 'touch'
 
-const ADDED = Symbol()
-const BUILDING = Symbol()
-const BUILT = Symbol()
+const ADDED = Symbol('added')
+const BUILDING = Symbol('building')
+const BUILT = Symbol('built')
```

Comment:
This is just a description for debugging purposes. See https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Global_Objects/Symbol